cadence/templates/web-ingress.yaml

```diff
@@ -1,7 +1,5 @@
 {{- if .Values.web.ingress.enabled -}}
-{{- $serviceName := include "cadence.fullname" . -}}-web
-{{- $servicePort := .Values.web.service.port -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Can you please add a conditional check if the API version is available?

Reply:
Thanks for the review @sagikazarmark 

I've added a conditional check for the `apiVersion` now. Please check again.